portfolio/src/main/webapp/index.html

```diff
@@ -51,9 +51,11 @@ <h3>Past entries</h3>
         <option value=10>10</option>
         <option value=15>15</option>
       </select>
-      <label for="language">What language?</label>
+      <label for="language">  What language?</label>
```

Comment:
Why the space here?

Reply:
It spaces out the two selectors - without it, they are right next to each other on the page